evm.md

```diff
@@ -322,8 +322,8 @@ The `#next [_]` operator initiates execution by:
     rule <k> #next [ OP ] => #end EVMC_STATIC_MODE_VIOLATION ... </k>
          <wordStack> WS </wordStack>
          <static> STATIC:Bool </static>
-      requires notBool ( #stackUnderflow(WS, OP) orBool #stackOverflow(WS, OP) )
-       andBool STATIC andBool #changesState(OP, WS)
+      requires STATIC andBool #changesState(OP, WS)
+       andBool notBool ( #stackUnderflow(WS, OP) orBool #stackOverflow(WS, OP) )
```

Comment:
Is this because `STATIC andBool #changesState(OP, WS)` is faster to check?

Reply:
Well I noticed this group of rules taking a long time overall, especially #stackOverflow, so it seemed reasonable to me to check it last